src/ui_ng/lib/src/helm-chart/versions/helm-chart-version.component.html

```diff
@@ -0,0 +1,133 @@
+                    </span>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="row">
+        <div *ngIf="!isCardView" class="col-lg-12 col-md-12 col-sm-12 col-xs-12">
+            <clr-datagrid (clrDgRefresh)="refresh($event)" [clrDgLoading]="loading" [(clrDgSelected)]="selectedRows">
+                <clr-dg-action-bar>
+                    <button type="button" class="btn btn-sm btn-secondary"
+                    [disabled]="!hasProjectAdminRole"
+                    (click)="versionUpload($event)">{{'HELM_CHART.UPLOAD' | translate}}</button>
+                    <button type="button" class="btn btn-sm btn-secondary"
+                    [disabled]="!(selectedRows.length===1)"
+                    (click)="versionDownload()">
+                        <clr-icon shape="times" size="16"></clr-icon>&nbsp;{{'HELM_CHART.DOWNLOAD' | translate}}</button>
+                    <button type="button" class="btn btn-sm btn-secondary"
+                    [disabled]="selectedRows.length<=0 || !hasProjectAdminRole"
+                    (click)="openVersionDeleteModal(selectedRows)">
+                        <clr-icon shape="times" size="16"></clr-icon>&nbsp;{{'BUTTON.DELETE' | translate}}</button>
+                </clr-dg-action-bar>
+                <clr-dg-column [clrDgField]="'name'">{{'HELM_CHART.VERSION' | translate}}</clr-dg-column>
+                <clr-dg-column>{{'HELM_CHART.ENGINE' | translate }}</clr-dg-column>
+                <clr-dg-column>{{'HELM_CHART.MAINTAINERS' | translate }}</clr-dg-column>
+                <clr-dg-column>{{'HELM_CHART.CREATED' | translate }}</clr-dg-column>
+                <clr-dg-row *ngFor="let v of chartVersions" [clrDgItem]="v">
+                    <clr-dg-cell>
+                        <span class="list-img"><img [src]="getImgLink(v)"/></span>
+                        <a href="javascript:void(0)" (click)="onVersionClick(v)">{{ v.version }}</a>
+                    </clr-dg-cell>
+                    <clr-dg-cell>{{ v.engine }}</clr-dg-cell>
+                    <clr-dg-cell>{{ getMaintainerString(v.maintainers) }}</clr-dg-cell>
+                    <clr-dg-cell>{{ v.created | date}}</clr-dg-cell>
+                </clr-dg-row>
+                <clr-dg-footer>
+                    <clr-dg-pagination #pagination [clrDgPageSize]="pageSize">
+                        {{pagination.firstItem + 1}} - {{pagination.lastItem + 1}} {{'HELM_CHART.OF' | translate}} {{pagination.totalItems}} {{'HELM_CHART.VERSIONS'
+                        | translate}}
+                    </clr-dg-pagination>
+                </clr-dg-footer>
+            </clr-datagrid>
+        </div>
+    </div>
+    <div *ngIf="isCardView" class="row card-container">
+        <div *ngFor="let item of chartVersions;" class="col-lg-3 col-md-4 col-sm-6">
+            <a let i=index; class="card clickable" (click)="onVersionClick(item)">
+                <div class="card-header">
```

Comment:
Please use header ariticle footer instead.

Reply:
`card-header` is a class name defined by clarity